indigo_app/templates/place/tabbed_layout.html

```diff
@@ -62,6 +62,11 @@
             <i class="fas fa-fw fa-cog"></i> {% trans "Settings" %}
           </a>
         {% endif %}
+        {% if user_can_change_place_settings %}
```

Comment:
we _might_ want a different permission for this, but I'm not too sure. just noting for now to come back to.

Reply:
What is the permission used in the existing badge view where it can be granted on a user-by-user basis?

Reply:
the permission required for badges is `auth.change_user`,

Reply:
`auth.change_user`: https://github.com/laws-africa/indigo/blob/master/indigo_social/views.py#L40